### PR TITLE
import updates 2

### PR DESCRIPTION
**Pull request by imbstack**

### CODE REVIEW

File: scripts/importers/finish.sql [hunk 1]
```diff
@@ -12,6 +11,12 @@ alter table newsletters add constraint uix_email_name unique (email_id, "name");
 alter table vpn_waitlist add constraint vpn_waitlist_email_id_key unique (email_id);
 alter table fxa add constraint fxa_fxa_id_key unique (fxa_id);
 
+-- Easy enough to add this now too
+create index bulk_read_index on emails (update_timestamp, email_id);
+
+-- New matching
+CREATE UNIQUE INDEX idx_email_primary_unique_email_lower ON public.emails USING btree (lower((primary_email)::text));
```
Comment (jwhitlock) [inline scripts/importers/finish.sql:18]: Thanks for the reminder, I need to do some model and index cleanup. Filed https://github.com/mozilla-it/ctms-api/issues/238.

File: scripts/importers/finish.sql [hunk 2]
```diff
@@ -44,3 +49,4 @@ alter table mofo add constraint mofo_email_id_fkey foreign key (email_id) refere
 
 -- One more cleanup
 update newsletters set "format" = 'T' where "format" = 'N';
+alter table newsletters drop column source, add column source text;
```
Comment (jwhitlock) [inline scripts/importers/finish.sql:52]: What was the purpose of this cleanup?

**Review by jwhitlock — APPROVED** [review]
Thanks @imbstack!

File: scripts/importers/finish.sql (hunk 2 above)
Comment (imbstack) [reply scripts/importers/finish.sql:52]: That was the empty-string for source instead of null cleanup

**Pull request merged**